actions/wait-for-workflow-job/action.yml

```diff
@@ -108,7 +108,7 @@ runs:
 
             # Fetch the job details from the GitHub API
             job_details=$(gh api -H "Accept: application/vnd.github.v3+json" \
-                "https://api.github.com/repos/${REPO}/actions/jobs/$job_id")
```

Comment:
For this [workflow](https://github.com/smartcontractkit/.github/actions/runs/9807182065/job/27080419808), I'm getting results on https://api.github.com/repos/smartcontractkit/.github/actions/jobs/27080419808 but not https://api.github.com/repos/smartcontractkit/.github/actions/runs/9807182065/jobs/27080419808. Is this right?

Reply:
hey @manjunathb4461 -> all good, i was missing something.

Reply:
This was a test PR? okay

There's also `create draft pull request` button, that'll avoid automatically assigning people